digdag-core/src/main/java/io/digdag/core/log/LogFiles.java

```diff
@@ -62,6 +63,16 @@ public static String formatFileName(String taskName, Instant firstLogTime, Strin
                 agentId) + LOG_GZ_FILE_SUFFIX;
     }
 
+    public static String formatPlainTextFileName(String taskName, Instant firstLogTime, String agentId)
+    {
+        return String.format(ENGLISH,
```

Comment:
This code is duplicated with `formatFileName()`. How about extracting this `String.format()` to a private method and calling it from `formatFileName()` and `formatPlainTextFileName()`?

Reply:
I added generic private function that is `formatFileNameWithSuffix` and then both two methods to be used it.
I made it to be used